lib/ui/views/tokens_detail/layouts/components/token_detail_menu.dart

```diff
@@ -132,7 +135,41 @@ class TokenDetailMenu extends ConsumerWidget {
                       .scale(duration: const Duration(milliseconds: 250)),
                 ),
                 SizedBox(
-                  width: constraints.maxWidth * 0.33,
+                  width: constraints.maxWidth * 0.25,
+                  child: ActionButton(
+                    key: const Key('swapButton'),
+                    text: localizations.swapHeader,
+                    icon: aedappfm.Iconsax.arrange_circle_2,
+                    onTap: () async {
+                      sl.get<HapticUtil>().feedback(
+                            FeedbackType.light,
+                            preferences.activeVibrations,
+                          );
+
+                      final params = {
+                        'from': aeToken.isUCO ? 'UCO' : aeToken.address,
```

Comment:
Je crois qu'on a desormais une constante pour l'addresse de token `UCO`.

Reply:
not in the general aeToken management. some updates from aeSwap haven't been reported for the moment. We should work on it when we will create a common aeSwap SDK